Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseAutonomous.java

```diff
@@ -248,45 +307,192 @@ public void onError(int errorCode) {}
-            centerConeStack(robotCameraPipeline);
-            sleep(300);
-            //wait for grabber to close
+            centerConeStack(robotCameraPipeline, 300);
+            //close grabber
             driveGrabber(Constants.GRABBER_CLOSE_POSITION);
-            sleep(600);
+            //wait for grabber to close
+            sleep(500);
+            driveAutonomous(180, 0.1);
             //drive slides to stow position
-            driveSlides(Constants.SLIDE_LOW);
+            sleep(100);
+            driveSlidesAutonomous(Constants.SLIDE_LOW);
             //drive backwards 34.5 inches
-            driveAutonomous(180, 34.5);
+            driveAutonomous(180, 34.7);
+            sleep(100);
             //turn towards junction
-            turnToAngle(90 + angleOffset);
+            turnToAngle(90+angleOffset);
             //drive slides up
-            driveSlides(Constants.SLIDE_TOP);
+            driveSlidesAutonomous(Constants.SLIDE_HIGH-10);
+            sleep(100);
             //wait for slides to go all the way up
-            sleep(600);
             //drive forward
             driveAutonomous(0, 2);
-            //center on junction
-            centerJunctionTop(grabberCameraPipeline);
             sleep(100);
             //lower slides onto junction
-            driveSlides(Constants.SLIDE_HIGH - 100);
+            driveSlidesAutonomous(Constants.SLIDE_HIGH-300);
+            //open the grabber
+            driveGrabber(Constants.GRABBER_OPEN_POSITION);
+            //wait for cone to drop
+            sleep(100);
+            //drive slides back up
+            driveSlidesAutonomous(Constants.SLIDE_HIGH);
+            //drive backwards
+            driveAutonomous(180, 2);
+            //turn back to 0 heading
+            turnToAngle(0+angleOffset);
+        }
+    }
+    public void grabFromStackAndDepositOnJunctionPlusConeCentering(int loopNumber, int angleOffset){
+        blinkinChassis.setPattern(RevBlinkinLedDriver.BlinkinPattern.BLACK);
+        for (int i = 4; i >= 5 - loopNumber; i--) {
+            telemetry.addData("loop", i + 1);
+            turnToAngle(0 + angleOffset);
+            //drive slides to stack position
+            driveSlidesAutonomous(Constants.STACK_HEIGHTS[i]);
+            //center on cone stack
+            centerConeStack(robotCameraPipeline, 380);
+            //close grabber
+            driveGrabber(Constants.GRABBER_CLOSE_POSITION);
+            //wait for grabber to close
+            sleep(500);
+            driveAutonomous(180, 0.1);
+            //drive slides to stow position
             sleep(100);
+            driveSlidesAutonomous(Constants.SLIDE_LOW);
+            //drive backwards 34.5 inches
+            driveAutonomous(180, 34.7);
+            sleep(100);
+            //turn towards junction
+            turnToAngle(90 + angleOffset);
+            //drive slides up
+            driveSlidesAutonomous(Constants.SLIDE_HIGH - 10);
+            //:TODO EXPERIMENTAL
```

Comment:
Is there any way you can call `grabFromStackAndDepositOnJunction` in this method then just include the new stuff?